Fix newly broken test

### Problem

<!-- Brief description of issue -->
Even though tests were passing in the PR after #10213 was merged the tests are failing elsewhere.
https://github.com/internetarchive/openlibrary/actions/runs/12841310878/job/35810979725

> FAILED openlibrary/catalog/utils/__init__.py::openlibrary.catalog.utils.format_languages

<!-- Evidence, Screenshots, & Relevant URL(s) -->


### Reproducing the bug

1. Go to ...
2. Do ...

* Expected behavior:
* Actual behavior:


### Context

- Browser (Chrome, Safari, Firefox, etc):
- OS (Windows, Mac, etc):
- Logged in (Y/N):
- Environment (prod, dev, local): prod


### Breakdown

<!-- Overview of the proposed solution / implementation? -->

#### Requirements Checklist
* [ ]

#### Related files
<!-- Files related to this issue; this is super useful for new contributors who might want to help! If you're not sure, leave this blank; a maintainer will add them. -->
*

#### Stakeholders
<!-- @ tag stakeholders of this feature w/ questions or context as necessary -->
* @scottbarnes 
<hr>

#### Instructions for Contributors
<!-- Please leave the following reminder section as is to help new contributors and add instructions where necessary -->

- Please [run these commands](https://github.com/internetarchive/openlibrary/wiki/Git-Cheat-Sheet#working-on-your-branch) to ensure your repository is up to date **before** [creating a new branch](https://github.com/internetarchive/openlibrary/wiki/Git-Cheat-Sheet#making-changes-and-creating-a-pull-request) to work on this issue and **each time after** pushing code to Github, because the pre-commit bot may add commits to your PRs upstream.

--- a/openlibrary/catalog/utils/__init__.py
+++ b/openlibrary/catalog/utils/__init__.py
@@ -448,7 +448,7 @@ class InvalidLanguage(Exception):
 def format_languages(languages: Iterable) -> list[dict[str, str]]:
     """
     Format language data to match Open Library's expected format.
-    >>> format_languages(["eng", "fre"])
+    For an input of ["eng", "fre"], return:
     [{'key': '/languages/eng'}, {'key': '/languages/fre'}]
     """
     if not languages:

